frontend-service/src/components/SaveBar.js

```diff
@@ -0,0 +1,36 @@
+import { useState, useEffect, forwardRef} from "react";
+
+import MuiAlert from '@mui/material/Alert';
+import Snackbar from '@mui/material/Snackbar';
+
+
+const Alert = forwardRef(function Alert(props, ref) {
+  return <MuiAlert elevation={6} ref={ref} variant="filled" {...props} />;
+});
+
+const SaveBar = (props) => {
+  const [open, setOpen] = useState(false);
```

Comment:
If you are already taking in an `isOpen` variable in the props, why store it as a state also? I suggest taking in two props - `isOpen` and `onClose`.